pkg/kubestatemetrics/store/store.go

```diff
@@ -201,11 +201,9 @@ func (s *MetricsStore) Push(familyFilter FamilyAllow, metricFilter MetricAllow)
 				if !metricFilter(metric) {
 					continue
 				}
-				tags := metric.Labels
-				tags["uid"] = string(u)
```

Comment:
From unit tests, it appears this was done on purpose (though I do agree with the change, I don't see why we should report uid as tags), any idea why?

Reply:
I had no idea why we added the `uid` as a tag, before making this change I asked @CharlyF since he was the one who worked on the store part and he confirmed there is no good reason to keep it 😅